src/exam/Exam.jsx

```diff
@@ -51,6 +74,14 @@ const Exam = ({ isTimeLimited, children }) => {
 
   return (
     <div className="d-flex flex-column justify-content-center">
+      {shouldShowMasqueradeAlert() && (
+        <Alert variant="info" icon={Info} data-testid="masquerade-alert">
+          <FormattedMessage
+            id="exam.hiddenContent"
```

Comment:
Do we need this message to be specified somewhere for translation?

Reply:
Nope, `FormattedMessage` takes care of that